src/quo/components/wallet/network_routing/view.cljs

```diff
@@ -172,6 +173,16 @@
             [rn/view {:style (style/max-limit-bar-background network-name)}]
             [dashed-line network-name]])]))))
 
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:networks [:maybe :coll-of :any]]
+      [:container-style [:maybe :map]]
```

Comment:
Wherever you use `:maybe` I think we'd also want a `{:optional true}` to cover the cases when the prop is not passed at all. `:maybe` just covers the case when it's `nil`